src/callbacks/onResize/editYAxisTicks.js

```diff
@@ -33,4 +33,15 @@ export default function addYAxisTicks() {
         //Draw the gridlines.
         this.drawGridlines();
     }
+
+    //Draw custom y-axis given a log scale.
+    if (this.config.y.type === 'log') {
```

Comment:
looks much better but some weird stuff happening. I'm seeing that Direct Bilirubin has an 000004 guy, or ALT has repeated 0.0s, and temperature doesnt have an axis @samussiah I was playing with it some too -> could be worth changing the format - im just not sure why the 0.0s and no axis ones are happening